### PR TITLE
Stage 1 futurize on SM

### PR DESCRIPTION
**Pull request by MarkSymsCtx**
Mechanical code update using

futurize --stage1

https://python-future.org/futurize.html

Signed-off-by: Mark Syms <mark.syms@citrix.com>

### CODE REVIEW

**Review by TimSmithCtx — APPROVED** [review]

File: tests/test_mpathcount.py [hunk 1]
```diff
@@ -1,6 +1,7 @@
 """
 Unit tests for mpathcount
 """
+from __future__ import print_function
```
Comment (BenSimsCitrix) [inline tests/test_mpathcount.py:4]: Doesn't do any harm but any idea why its import future here and not used it.

Comment (BenSimsCitrix) [reply tests/test_mpathcount.py:4]: Ahh there a print on line 120, that must be why.

**Review by BenSimsCitrix — COMMENTED** [review]
LGTM

**Review by BenSimsCitrix — APPROVED** [review]
LGTM

**Pull request merged**